ext/debase_internals.c

```diff
@@ -371,7 +373,7 @@ process_line_event(VALUE trace_point, void *data)
       context->stop_frame = -1;
     }
 
-    breakpoint = breakpoint_find(breakpoints, path, lineno);
+    breakpoint = breakpoint_find(breakpoints, path, lineno, binding);
```

Comment:
I'd rather pass whole tracepoint object here for following reason:
binding calculation cost is super high, calling it on every line event will slow down debugging dramatically (more 2x) even if there are no conditional breakpoints. Thus getting binding should really be the latest operation before condition evaluation